package.json

```diff
@@ -241,7 +241,7 @@
     "@inquirer/editor": "^4.2.23",
     "@inquirer/input": "^4.3.1",
     "@inquirer/select": "^4.4.2",
-    "@modelcontextprotocol/sdk": "^1.22.0",
+    "@modelcontextprotocol/sdk": "1.22.0",
```

Comment:
consider if required

Reply:
So this one specifically was required.  I need to come back and open a bug on their repo, but the 1.23.0 release introduced some features that completely break TypeScript memory consumption.